plugins/filter/latest_version.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+
```

Comment:
Would it be better to name this `_latest_version` and state that it's private in the docs? In the longer term, I think this would be a good fit for `ansible.utils`.

Reply:
I don't really like the idea of plugins/modules starting with `_` showing up in the plugin/module list on the collection's docsite. Right now there's no way to mark a plugin properly as private so it won't show up in the docs (which includes `ansible-doc`). Standalone roles can have included plugins, but apparently that doesn't work for collection roles (at least it didn't work when I tried this in an earlier version of this PR); that would have been my preferred solution.

Reply:
that would have been my preferred solution too

Reply:
Ref: https://github.com/ansible/ansible/pull/79218 and https://github.com/ansible-community/community-topics/issues/154

Actually right now plugins starting with `_` are shown by `ansible-doc --list` as "deprecated" and are shown without the leading underscore. So renaming `latest_version` to `_latest_version` produces the following output:
```shell
$ ansible-doc --list --type filter community.sops
community.sops.decrypt         Decrypt sops encrypted data                                                                                                                               

DEPRECATED:
community.sops.latest_version  Get latest version from a list of versions              
```